package.json

```diff
@@ -60,5 +59,10 @@
   },
   "nx": {
     "includedScripts": []
+  },
+  "overrides": {
+    "@jscutlery/semver": {
+      "@nx/devkit": "19.2.2"
+    }
```

Comment:
This was required because, at this time, [semver doesn't support v19](https://github.com/jscutlery/semver/issues/814)